### DISCUSSION TITLE
Version 3.64 & Win/Mac一键安装包

### DISCUSSION THREAD

**Discussion by binary-husky** [binary-husky/gpt_academic#1404] [Announcements]
支持直接拖拽文件到上传区
支持将图片粘贴到输入区
修复若干隐蔽的内存BUG
修复多用户冲突问题
接入Deepseek Coder
AutoGen多智能体插件测试版
修复本地模型在Windows下的加载BUG
支持文心一言v4和星火v3
重构了前端UI和Gradio依赖

<hr /><em>This discussion was created from the release <a href='https://github.com/binary-husky/gpt_academic/releases/tag/'>Version 3.64 & Win/Mac一键安装包</a>.</em>

**Comment by Xu-Jam** [comment]
我还以为会接入gemini呢😂

**Comment by ahmtzzh** [comment]
一键安装版安装完该如何运行吖？
